changelog.md

```diff
@@ -1,3 +1,5 @@
+# future
+- [FIXED] MSSQL handle large bulk inserts
```

Comment:
Link to issue which is being fixed https://github.com/sequelize/sequelize/issues/6866